com.unity.render-pipelines.universal/Editor/2D/Renderer2DMenus.cs

```diff
@@ -111,62 +101,62 @@ static bool CreateLightValidation()
             return Light2DEditorUtility.IsUsing2DRenderer();
         }
 
-        [MenuItem("GameObject/Light/Freeform Light 2D/Square", false, k_MenuPriority)]
+        [MenuItem("GameObject/Light/Freeform Light 2D/Square", priority = CoreUtils.Sections.section3)]
         static void CreateSquareFreeformLight2D(MenuCommand menuCommand)
         {
             CreateLight(menuCommand, Light2D.LightType.Freeform, FreeformPathPresets.CreateSquare());
         }
 
-        [MenuItem("GameObject/Light/Freeform Light 2D/Circle", false, k_MenuPriority)]
+        [MenuItem("GameObject/Light/Freeform Light 2D/Circle", priority = CoreUtils.Sections.section3)]
         static void CreateCircleFreeformLight2D(MenuCommand menuCommand)
         {
             CreateLight(menuCommand, Light2D.LightType.Freeform, FreeformPathPresets.CreateCircle());
         }
 
-        [MenuItem("GameObject/Light/Freeform Light 2D/Isometric Diamond", false, k_MenuPriority)]
+        [MenuItem("GameObject/Light/Freeform Light 2D/Isometric Diamond", priority = CoreUtils.Sections.section3)]
         static void CreateIsometricDiamondFreeformLight2D(MenuCommand menuCommand)
         {
             CreateLight(menuCommand, Light2D.LightType.Freeform, FreeformPathPresets.CreateIsometricDiamond());
         }
 
-        [MenuItem("GameObject/Light/Freeform Light 2D/Hexagon Flat Top", false, k_MenuPriority)]
+        [MenuItem("GameObject/Light/Freeform Light 2D/Hexagon Flat Top", priority = CoreUtils.Sections.section3)]
         static void CreateHexagonFlatTopFreeformLight2D(MenuCommand menuCommand)
         {
             CreateLight(menuCommand, Light2D.LightType.Freeform, FreeformPathPresets.CreateHexagonFlatTop());
         }
 
-        [MenuItem("GameObject/Light/Freeform Light 2D/Hexagon Pointed Top", false, k_MenuPriority)]
+        [MenuItem("GameObject/Light/Freeform Light 2D/Hexagon Pointed Top", priority = CoreUtils.Sections.section3)]
         static void CreateHexagonPointedTopFreeformLight2D(MenuCommand menuCommand)
         {
             CreateLight(menuCommand, Light2D.LightType.Freeform, FreeformPathPresets.CreateHexagonPointedTop());
         }
 
-        [MenuItem("GameObject/Light/Sprite Light 2D", false, k_MenuPriority)]
+        [MenuItem("GameObject/Light/Sprite Light 2D", priority = CoreUtils.Sections.section3)]
         static void CreateSpriteLight2D(MenuCommand menuCommand)
         {
             CreateLight(menuCommand, Light2D.LightType.Sprite);
         }
 
-        [MenuItem("GameObject/Light/Spot Light 2D", false, k_MenuPriority)]
+        [MenuItem("GameObject/Light/Spot Light 2D", priority = CoreUtils.Sections.section3)]
         static void CreatePointLight2D(MenuCommand menuCommand)
         {
             CreateLight(menuCommand, Light2D.LightType.Point);
         }
 
-        [MenuItem("GameObject/Light/Global Light 2D", false, k_MenuPriority)]
+        [MenuItem("GameObject/Light/Global Light 2D",  priority = CoreUtils.Sections.section3)]
```

Comment:
Shouldn't we enforce an order here like for other sections ?

Reply:
I've enforce it for clarification, but the alphabetical order was correct too.